src/thread.rs

```diff
@@ -101,7 +101,7 @@ pub fn attach_permanently(jvm: JvmPtr) -> GlobalResult<AttachGuard> {
     })
 }
 
-pub unsafe fn attach<'jvm>(jvm: JvmPtr) -> GlobalResult<AttachGuard> {
```

Comment:
while we're here we should add a safety section to the docs or make this not pub